src/aspire/source/image.py

```diff
@@ -114,6 +115,7 @@ def as_copy(func):
     is best used on methods that mutate the object but don't return anything.
     """
 
+    @functools.wraps(func)  # Pass metadata (eg name and doctrings) from `func`
```

Comment:
This great. Docs look good.

Should we make this  `as_copy` private? Currently it appears in the docs.

Reply:
lol, I'd rather delete it, but sure, I'll make a commit with that.